app/models/policy_event/purging.rb

```diff
@@ -12,16 +12,6 @@ def purge_window_size
         ::Settings.policy_events.history.purge_window_size
       end
 
-      def purge_queue
-        MiqQueue.put_unless_exists(
-          :class_name  => name,
-          :method_name => "purge",
-          :role        => "event",
-          :queue_name  => "ems"
-        )
-      end
-      alias_method :purge_timer, :purge_queue
-
```

Comment:
Hmm, is the commit message wrong here?  

```
- converted put_or_update to put on general queue
- queuing directly into purge_by_date
```

Are we now using the mixin's method which does a put?  All I see is deleted lines here, no new method.